src/index.ts

```diff
@@ -95,6 +97,15 @@ export const htmlPlugin = (configuration: Configuration = { files: [], }): esbui
         }
     }
 
+    function renderTemplate(htmlFileConfiguration: HtmlFileConfiguration, htmlTemplate: string) {
+        const templateContext = {
+            define: htmlFileConfiguration.define,
+        };
+
+        const compiledTemplateFn = lodashTemplate(htmlTemplate)
+        return compiledTemplateFn(templateContext)
```

Comment:
Hattest du ein konkretes Ziel im Sinn für den extra scope "define" im Template-Kontext? Macht eventuell Sinn `define` direkt als Kontext zu übergeben? 
```suggestion
        return compiledTemplateFn(htmlFileConfiguration.define)
```

Reply:
In Zukunft macht es denke ich Sinn, z.B. Werte aus esbuild oder aus dem plugin selbst zu Verfügung zustellen, z.B.:

```
const templateContext = {
    define: define,
    esbuild: build, // esbuild build metadata
    htmlPlugin: metadata,
}
```